pinot-controller/src/test/java/org/apache/pinot/controller/recommender/realtime/provisioning/MemoryEstimatorTest.java

```diff
@@ -116,7 +117,7 @@ private void runTest(String schemaFileName, Consumer<String> assertFunc) throws
     assertFunc.accept(metadata);
 
     // cleanup
-    FileUtils.deleteDirectory(generatedSegment);
+    FileUtils.deleteDirectory(workingDir);
```

Comment:
The test should not have to do this. The estimator should internally delete the directory right since everything it generates is transient?

Reply:
Please refer to the next comment.